src/ukify/ukify.py

```diff
@@ -913,18 +952,39 @@ def config_example(self) -> tuple[Optional[str], Optional[str], Optional[str]]:
         help = 'OpenSSL engine to use for signing',
         config_key = 'UKI/SigningEngine',
     ),
+    ConfigItem(
+        '--signtool',
+        choices=('sbsign', 'pesign'),
+        dest='signtool',
+        default='sbsign',
+        help='whether to use sbsign or pesign. Default is sbsign.',
+        config_key = 'UKI/SecureBootSigningTool',
+    ),
     ConfigItem(
         '--secureboot-private-key',
         dest = 'sb_key',
-        help = 'path to key file or engine-specific designation for SB signing',
+        help = 'required by --signtool=sbsign. Path to key file or engine-specific designation for SB signing',
         config_key = 'UKI/SecureBootPrivateKey',
     ),
     ConfigItem(
         '--secureboot-certificate',
         dest = 'sb_cert',
-        help = 'path to certificate file or engine-specific designation for SB signing',
+        help = 'required by --signtool=sbsign. sbsign needs a path to certificate file or engine-specific designation for SB signing',
         config_key = 'UKI/SecureBootCertificate',
     ),
+    ConfigItem(
+        '--secureboot-certificate-dir',
+        dest='sb_certdir',
+        default='/etc/pki/pesign',
+        help='required by --signtool=pesign. Path to nss certificate database directory for PE signing. Default is /etc/pki/pesign',
+        config_key = 'UKI/SecureBootCertificateDir',
+    ),
+    ConfigItem(
+        '--secureboot-certificate-name',
+        dest='sb_cert_name',
+        help='required by --signtool=pesign. pesign needs a certificate nickname of nss certificate database entry to use for PE signing',
+        config_key = 'UKI/SecureBootCertName',
```

Comment:
Now we have two options, one with "Cert" and one with "Certificate". Please use "Certificate".